stemgen/nistemfile.py

```diff
@@ -23,16 +24,16 @@
 ]
 
 
-def _extract_cover(f):
-    tag = None
-    if isinstance(f, tagpy.FileRef):
-        tag = f.tag()
-        f = f.file()
```

Comment:
Unfortunately, you still will need to assert the instance type, as typing annotation are only indicative

Reply:
We are only passing `tagpy.FileRef` into the `_extract_cover` function, no?
```
src = tagpy.FileRef(src)
...
cover = _extract_cover(src)
```